### ISSUE TITLE
Cannot parse XML from DependencyCheck 5.3.0 (XSD 2.3)

### ISSUE THREAD

**Issue by TobiX** [dependency-check/dependency-check-sonar-plugin#223]
**Describe the bug**

When running analysis against an XML report from DependencyCheck 5.3.0 (new [XSD 2.3](https://github.com/jeremylong/DependencyCheck/blob/master/core/src/main/resources/schema/dependency-check.2.3.xsd)) the report just fails with

```
[WARNING] XML-Analysis aborted due to: Mandatory elements are missing. Plugin is compatible to https://jeremylong.github.io/DependencyCheck/dependency-check.2.2.xsd
```

(Yeah, I know, I need to switch to JSON - soon 😬)

**Expected behavior**
Maybe make the plugin compatible with multiple versions of the XML report? Not  serious problem for me, since I'm currently in the process of switching all my analysis to the JSON report...

**Versions (please complete the following information):**

-   dependency-check 5.3.0
-   sonarqube 7.9.2
-   dependency-check-sonar-plugin 2.0.2

**Comment by Reamer** [comment]
I don't want to actively support multiple versions of reports. In the event of errors, only the latest published version of Dependency-Check is supported.
Fixed by #230 and #229

**Comment by TobiX** [comment]
> I don't want to actively support multiple versions of reports.

That's unfortunate, but understandable. Thanks for fixing.
